docs/adr/ADR-004a-multi-hypervisor-implementation-details.md

```
@@ -0,0 +1,837 @@
+        val username: String,
+        val passwordEncrypted: ByteArray,
+        val managedSystem: String
+    ) : HypervisorConnectionParams() {
+        override val hypervisorType = HypervisorType.POWERVM
+    }
+}
+```
+
+### 4.2 Migration Strategy (Two-Phase)
+
+**Phase 1: Deprecation (Release N)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)